apps/hash-frontend/src/pages/shared/block-collection/component-view.tsx

```diff
@@ -199,7 +199,6 @@ export class ComponentView implements NodeView {
               <BlockCollectionContext.Consumer>
                 {(collectionContext) => (
                   <BlockLoader
-                    key={entityId} // reset the component state when the entity changes
```

Comment:
I don't think this is necessary any more and I believe it was contributing to flashes of newly created paragraphs when the `entityId` was received from the API.